api/core/v1beta2/machine_phase_types.go

```diff
@@ -45,6 +45,10 @@ const (
 	// become a Kubernetes Node in a Ready state.
 	MachinePhaseRunning = MachinePhase("Running")
 
+	// MachinePhaseUpdating is the Machine state when the Machine
+	// is updating.
```

Comment:
may be
```suggestion
	// is updating in-place.
```
because this is the only type of update we support at machine level